src/containers/education/Education.js

```diff
@@ -4,14 +4,17 @@ import EducationCard from '../../components/educationCard/EducationCard';
 import { educationInfo } from '../../portfolio';
 
 export default function Education() {
-  return (
-    <div className="education-section" id="education">
-        <h1 className="education-heading">Education</h1>
-        <div className="education-card-container">
-          {educationInfo.schools.map((school) => (
-            <EducationCard school={school} />
-          ))}
-        </div>
-    </div>
-  );
+   if(educationInfo.viewEducation){
```

Comment:
Avoid formatting as it shows changes in lines where there are none which can be confusing while reviewing. It isn't a problem for this small PR but can be really difficult in big ones.